presto-tests/src/main/java/com/facebook/presto/tests/TestingPrestoClient.java

```diff
@@ -193,6 +193,9 @@ else if (type instanceof VarcharType) {
         else if (VARBINARY.equals(type)) {
             return value;
         }
+        else if (type.toString().equals("ObjectId")) { // So there's no circular dependency (presto-mongodb -> presto-tests -> presto-mongodb)
```

Comment:
@martint https://gist.github.com/kfitzgerald/06fd13f8a3e5d72d15e0825ce83b4621

`TestMongoIntegrationSmokeTest.createTableWithEveryType`
(TestMongoIntegrationSmokeTest.java:71) 
* was updated to include an ObjectId type, which causes the test client to throw

and 

`TestMongoIntegrationSmokeTest.testObjectIds`
(TestMongoIntegrationSmokeTest.java:155)
* was added to prove that querying by ObjectId works, but throws for the same reason